sqlstore/src/main/resources/db/migration_beforeMigrate/05_add_platform.sql

```diff
@@ -1,8 +1,10 @@
+-- If you change these method names or signatures, update the docs in `docs/_posts/2017-12-07-value-type-data.md`
 -- StartNoTest
 DELIMITER //
 
 DROP PROCEDURE IF EXISTS addPlatform//
-CREATE PROCEDURE addPlatform(
+DROP PROCEDURE IF EXISTS addInstrumentModel//
+CREATE PROCEDURE addInstrumentModel(
```

Comment:
Rename breaks OICR after-migrate script, so should have a PR there as well: https://github.com/oicr-gsi/miso-lims/blob/oicr/sqlstore/src/main/resources/db/migration_afterMigrate/65_sequencers.sql

Reply:
OICR PR: https://github.com/oicr-gsi/miso-lims/pull/210